scripts/hid_configurator/configurator_cli.py

```diff
@@ -0,0 +1,142 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+import argparse
+import logging
+
+from configurator_core import DEVICE
+from configurator_core import get_device_pid, open_device, fwinfo, fwreboot, change_config, fetch_config, dfu_transfer
+
+
+def progress_bar(permil):
+    length = 40
+    done_len = length * permil // 1000
+    progress_line = '[' + '*' * done_len + '-' * (length - done_len) + ']'
+    percent = permil / 10.0
+    print('\r{} {}%'.format(progress_line, percent), end='')
+
+
+def perform_dfu(dev, args):
+    dfu_image = args.dfu_image
+    recipient = get_device_pid(args.device_type)
+
+    success = dfu_transfer(dev, recipient, dfu_image, progress_bar)
+
+    if success:
+        print('DFU transfer completed')
+        return True
+    else:
+        print('DFU transfer failed')
+        return False
+
+
+def perform_config(dev, args):
+    module_config = DEVICE[args.device_type]['config'][args.module]
+    module_id = module_config['id']
+    device_options = module_config['options']
+
+    config_name = args.option
+    recipient = get_device_pid(args.device_type)
+
+    if args.value is None:
+        success, val = fetch_config(dev, recipient, config_name, device_options, module_id)
+        if success:
+            print('Fetched {}: {}'.format(config_name, val))
+        else:
+            print('Failed to fetch {}'.format(config_name))
+    else:
+        config_value = int(args.value)
+        success = change_config(dev, recipient, config_name, config_value, device_options, module_id)
+
+        if success:
+            print('{} set to {}'.format(config_name, config_value))
+        else:
+            print('Failed to set {}'.format(config_name))
+
+
+def perform_fwinfo(dev, args):
+    recipient = get_device_pid(args.device_type)
+    info = fwinfo(dev, recipient)
+    if info:
```

Comment:
please add one empty line before info to be in line with other functions

Reply:
done